src/microbe_stage/Microbe.cs

```diff
@@ -1535,26 +1684,84 @@ private void ReadyToReproduce()
         }
     }
 
+    /// <summary>
+    ///   Removes the player the ability to go to the editor.
+    ///   Does nothing when called by the AI.
+    /// </summary>
+    private void UnreadyToReproduce()
+    {
+        if (!IsPlayerMicrobe)
+            return;
+
+        // allOrganellesDivided = false;
+        OnReproductionStatus?.Invoke(this, false);
```

Comment:
Other than the player, cells don't have this callback anyway, so I don't see the need to have the !IsPlayerMicrobe guard here.